_api_app/app/Sites/Sections/SiteSectionTagsDataService.php

```diff
@@ -26,7 +27,7 @@
  * </sections>
  * ```
  */
-class Tags Extends Storage {
+class SiteSectionTagsDataService Extends Storage {
```

Comment:
I think we can simplify this a bit, by using just one level above in hierarchy - `SectionTagsDataService`. A bit shorter name but also includes reference to what it belongs. What do you say @uldisrudzitis ?

Reply:
@vteivans 👍